tests/integration/test_data/gomod_e2e_multiple_modules/bom.json

```diff
@@ -196,17 +196,6 @@
       "purl": "pkg:golang/internal/goarch?type=package",
       "type": "library"
     },
-    {
```

Comment:
I wonder if these changes aren't better suited to be part of https://github.com/containerbuildsystem/cachi2/pull/427/commits/6ec74a0243c56d0fee175acf1a49c6a59f10c66b rather than a standalone commit, I think that would make more sense.

Reply:
Yes, they should, moved...